This syntax is not supported in packaged rollup/node yet.
Minified file is generated by uglifyjs.terser command line.

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -1,4 +1,3 @@
-import {terser} from "rollup-plugin-terser";
 import * as meta from "./package.json";
 
 const config = {
@@ -17,20 +16,5 @@
 };
 
 export default [
-  config,
-  {
-    ...config,
-    output: {
-      ...config.output,
-      file: `dist/${meta.name}.min.js`
-    },
-    plugins: [
-      ...config.plugins,
-      terser({
-        output: {
-          preamble: config.output.banner
-        }
-      })
-    ]
-  }
+  config
 ];
